jesthtmlreporter.config.json

```diff
@@ -0,0 +1,5 @@
+{
+    "pageTitle": "RingCentral Webphone Unit Test",
+    "outputPath": "test_report/index.html",
+    "includeFailureMsg": true
+}
```

Comment:
Jest (with Puppeteer) is already used in this project, is this test addition compatible with the existing tests? Can you make sure they coexist properly?

Reply:
The UT framework is actually added by Elias. I'll have a look if we can remove it and use the existed one

Reply:
No need to remove it. Make them coexist.